webhook/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "pelias-update-webhook",
+  "version": "1.0.0",
```

Comment:
Let's add semantic-release like in https://github.com/ibi-group/pelias-stitch/pull/1 and https://github.com/ibi-group/pelias-stitch/pull/2 so that this version number gets updated.

Reply:
Added in dbab857d611c1d6aaf18033fba41217c69a193e2 and to ci in 6063c2699218b70b816e4ae1d4c1e7a96488038b